et_replay/utils.py

```diff
@@ -6,7 +6,7 @@
 import uuid
 from typing import Any, Dict
 
-from et_replay.lib.execution_trace import ExecutionTrace
+from et_replay.execution_trace import ExecutionTrace
```

Comment:
nit: for the record, this can also be `from et_replay import ExecutionTrace`. But this works too. (y)

Reply:
Thank you! This makes sense, updated.